homeassistant/components/lutron/light.py

```diff
@@ -54,14 +51,6 @@ class LutronLight(LutronDevice, LightEntity):
     _prev_brightness: int | None = None
     _attr_name = None
 
-    @property
-    def brightness(self) -> int:
-        """Return the brightness of the light."""
-        new_brightness = to_hass_level(self._lutron_device.last_level())
```

Comment:
If you don't call last_level you may not be getting the right value.  State is easily lost in a lutron system due to how the updates flow related to scenes.  I strongly discourage this approach as this has been stable for a long time now.